core/state/snapshot/disklayer_test.go

```diff
@@ -432,4 +435,81 @@ func TestDiskPartialMerge(t *testing.T) {
 // This test case is a tiny specialized case of TestDiskPartialMerge, which tests
 // some very specific cornercases that random tests won't ever trigger.
 func TestDiskMidAccountPartialMerge(t *testing.T) {
+	// TODO(@karalabe) ?
+}
+
+func tempDB() (ethdb.Database, error) {
```

Comment:
Nit, please inline this into the test, there's only one use for it, so might as well have one less dangling helper method. That will allow to `os.RemoveAll` the folder after you're done. It's super annoying when tests leave their files on disk after they are done.